src/backend/prisma/schema.prisma

```diff
@@ -53,6 +58,9 @@ model User {
   email                   String           @unique
   emailId                 String?          @unique
   role                    Role             @default(GUEST)
+  userSettingsId          String?          @unique
+  userSettings            UserSettings?    @relation(fields: [userSettingsId], references: [id])
```

Comment:
I think that the user should have just the userSettings object and the userSettings model should be concerned with the id and @relation tag. I don't know if it matters but it makes more sense to me in the sense that the user settings is then tied to the user

Reply:
Lemme experiment and I'll see if it's possible, I can't remember what did / didn't work

Reply:
Like this?

<img width="737" alt="Screen Shot 2022-07-02 at 3 40 26 AM" src="https://user-images.githubusercontent.com/51571627/176997238-444dd99d-c9af-4e35-8d08-5a82ec4d8f3d.png">

Reply:
yeee